docs/content/en/docs/getting-started/production-environment/baremetal-getstarted.md

```diff
@@ -166,26 +166,42 @@ Follow these steps if you want to use your initial cluster to create and manage
-   
-   ##### With hardware CSV
-   ```bash
-   eksctl anywhere create cluster \
-       -f eksa-w01-cluster.yaml  \
-       # --install-packages packages.yaml \ # uncomment to install curated packages at cluster creation
-       --hardware-csv <hardware.csv>
-       # --bundles-override ./eks-anywhere-downloads/bundle-release.yaml \ # uncomment for airgapped install
-       --kubeconfig mgmt/mgmt-eks-a-cluster.kubeconfig
-   ```
-   ##### Without hardware CSV
-   ```bash
-   eksctl anywhere create cluster \
-       -f eksa-w01-cluster.yaml  \
-       # --install-packages packages.yaml \ # uncomment to install curated packages at cluster creation
-       # --bundles-override ./eks-anywhere-downloads/bundle-release.yaml \ # uncomment for airgapped install
-       --kubeconfig mgmt/mgmt-eks-a-cluster.kubeconfig
-   ```
 
-   As noted earlier, adding the `--kubeconfig` option tells `eksctl` to use the management cluster identified by that kubeconfig file to create a different workload cluster.
+   Create a workload cluster in one of the following ways:
+   * eksctl CLI: To create a workload cluster with eksctl, run:
+
+     ##### With hardware CSV
+     ```bash
+     eksctl anywhere create cluster \
+         -f eksa-w01-cluster.yaml  \
+         # --install-packages packages.yaml \ # uncomment to install curated packages at cluster creation
+         --hardware-csv <hardware.csv>
+         # --bundles-override ./eks-anywhere-downloads/bundle-release.yaml \ # uncomment for airgapped install
+         --kubeconfig mgmt/mgmt-eks-a-cluster.kubeconfig
+     ```
+     ##### Without hardware CSV
+     ```bash
+     eksctl anywhere create cluster \
+         -f eksa-w01-cluster.yaml  \
+         # --install-packages packages.yaml \ # uncomment to install curated packages at cluster creation
+         # --bundles-override ./eks-anywhere-downloads/bundle-release.yaml \ # uncomment for airgapped install
+         --kubeconfig mgmt/mgmt-eks-a-cluster.kubeconfig
+     ```
+   * kubectl CLI: The cluster lifecycle feature lets you use kubectl to talks to the Kubernetes API to create a workload cluster. To use kubectl, run:
+      ```bash
+      kubectl apply -f eksa-w01-cluster.yaml --kubeconfig mgmt/mgmt-eks-a-cluster.kubeconfig
+      ```
+     As noted earlier, adding the `--kubeconfig` option tells `eksctl` to use the management cluster identified by that kubeconfig file to create a different workload cluster.
+
+   * **GitOps**: See [Manage separate workload clusters with GitOps]({{< relref "../../tasks/cluster/cluster-flux.md#manage-separate-workload-clusters-using-gitops" >}})
+
+   * **Terraform**: See [Manage separate workload clusters with Terraform]({{< relref "../../tasks/cluster/cluster-terraform.md#manage-separate-workload-clusters-using-terraform" >}})
+
+     >**NOTE**:For kubectl, GitOps and Terraform:
+     > 
+     > You can't add additional hardware when creating workload cluster.    
```

Comment:
We should not say users can't add additional hardware. Here we need to specify the steps to add the hardware manifests they can create using `eksctl anywhere generate hardware -z hardware.csv > hardware.yaml`. Once they have these manifests, they can apply the hardware.yaml to the mgmt cluster using `kubectl apply -f hardware.yaml`.